raphtory/src/db/api/view/edge.rs

```diff
@@ -432,7 +436,7 @@ mod test_edge_view {
 
         // FIXME: boolean properties not supported yet (Issue #48)
```

Comment:
I should have fixed this one :)

Reply:
Turns out it is not fixed because of secondary index problems (see #1780)